tests/data/model-validation.ts

```diff
@@ -804,15 +801,12 @@ type org
 `,
     expectedError: [
       {
-        endColumn: 22,
+        endColumn: 21,
         endLineNumber: 6,
-        message: "Assignable relation 'member' must have types",
-        relatedInformation: {
-          type: "assignable-relation-must-have-type",
-        },
+        message: "Invalid syntax",
```

Comment:
Same here

Reply:
See above - this is byproduct of grammar change which requires the types to be specified.